src/frontend/src/app/services/user.service.ts

```diff
@@ -7,9 +8,79 @@ import { USERS } from '../models/mock-users';
 })
 export class UserService {
 
-  constructor() { }
+  // constructor() { }
+
+  // getUsers(): User[] {
+  //   return USERS;
+  // }
```

Comment:
If this is pulling user objects from the backend, I would remove these commented lines to keep things clean!

Reply:
I believe these lines are coded by Joe and so I am not sure whether to remove it or not hence I commented these lines for time being.